src/coreComponents/physicsSolvers/LinearSolverParameters.cpp

```diff
@@ -257,6 +260,79 @@ void LinearSolverParametersInput::postProcessInput()
                         ": Invalid value." );
 
   // TODO input validation for other AMG parameters ?
+
+  if( getLogLevel() > 0 )
+    print();
+}
+
+void LinearSolverParametersInput::print()
+{
+  TableData tableData;
+  tableData.addRow( "Log level", getLogLevel());
+  tableData.addRow( "Linear solver type", EnumStrings< LinearSolverParameters::SolverType >::toString( m_parameters.solverType ) );
+  tableData.addRow( "Preconditioner type", EnumStrings< LinearSolverParameters::PreconditionerType >::toString( m_parameters.preconditionerType ) );
```

Comment:
The casts may be simplified, as a formatter exists in `Format.hpp`:
```suggestion
  tableData.addRow( "Linear solver type", m_parameters.solverType );
  tableData.addRow( "Preconditioner type", m_parameters.preconditionerType );
```
(there are a lot more below)